Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -51,17 +51,27 @@ public class RoslynTargetsTests
         public void Settings_ValidSetup_ForAnalyzedProject(string msBuildLanguage, bool isTestProject, string excludeTestProjects)
         {
             // Arrange and Act
-            var result = Execute_Settings_ValidSetup(msBuildLanguage, isTestProject, excludeTestProjects);
+            var dir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
```

Comment:
This is a detail and we can consider doing this maybe in another PR:
I'd propose to turn `CreateTestSpecificFolder` in `TestUtils` to public and use that one directly here, 
because using `CreateTestSpecificFolderWithSubPaths` for a single-level directory is a bit misleading.